### PR TITLE
Front end fix

### PR DESCRIPTION
**Pull request by BohdanBusko**
**STATUS**
READY

**DESCRIPTION**

screenshot from 2018-11-23 13-32-37
screenshot from 2018-11-23 13-32-11
screenshot from 2018-11-23 13-33-22

**CHECK LIST**
- [ ] specs added
- [ ] specs passed
- [ ] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs

### CODE REVIEW

File: app/assets/stylesheets/search.scss [hunk 1]
```diff
@@ -1,39 +1,119 @@
...
 #search-form {
+  margin-right: 10px;
   position: relative;
 
-  #search {
-    width: 260px;
+  .search-box {
+
+    &::after,
+    &::before {
```
Comment (pereslop) [inline app/assets/stylesheets/search.scss]: the same styles, than at app/assets/stylesheets/header.scss, can you make them reusable?

File: app/views/shared/_navbar_user_messages_counter.html.slim [hunk 2]
```diff
@@ -1,5 +1,5 @@
 - if current_user.messages.unreaded.count > 0
   .badge.is-badge-small.is-badge-outlined[data-badge="#{current_user.messages.unreaded.count}"]
-    = current_user.full_name
+    = current_user.full_name.truncate(15)
```
Comment (igorkasyanchuk) [inline app/views/shared/_navbar_user_messages_counter.html.slim]: use helper, truncate

Comment (igorkasyanchuk) [reply app/views/shared/_navbar_user_messages_counter.html.slim]: and never use this approach in views

Comment (BohdanBusko) [reply app/views/shared/_navbar_user_messages_counter.html.slim]: i use it

Comment (BohdanBusko) [reply app/views/shared/_navbar_user_messages_counter.html.slim]: oh or you mean rails helper because I use ruby method

**Review by igorkasyanchuk — APPROVED** [review]

**Pull request merged**